components/dash-core-components/package.json

```diff
@@ -46,9 +46,10 @@
     "fast-isnumeric": "^1.1.4",
     "file-saver": "^2.0.5",
     "highlight.js": "^11.4.0",
+    "mathjax": "3.2.0",
```

Comment:
https://www.npmjs.com/package/mathjax/v/3.2.0
Perhaps we could change & relax this to "^3.0.0"?

Reply:
We don't really package DCC for anyone else to use at the JS level - so we've generally gone with `^latest` - ie `^3.2.0` in this case.

Reply:
Done in 8ba1dcf8b22e251d7f77f020a6b9204246e8744d.